src/adapter/src/catalog/builtin.rs

```diff
@@ -1758,7 +1759,8 @@ pub static MZ_CLUSTER_REPLICAS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
         .with_column("cluster_id", ScalarType::String.nullable(false))
         .with_column("size", ScalarType::String.nullable(true))
         .with_column("availability_zone", ScalarType::String.nullable(true))
-        .with_column("owner_id", ScalarType::String.nullable(false)),
+        .with_column("owner_id", ScalarType::String.nullable(false))
+        .with_column("disk", ScalarType::Bool.nullable(true)),
```

Comment:
```suggestion
        .with_column("disk", ScalarType::Bool.nullable(false)),
```

Reply:
for unmanaged clusters this is null

Reply:
In `mz_clusters`, yeah, it's gotta be nullable to indicate "unmanaged". But for `mz_cluster_replicas` it should always be set, right? Like a replica always either has disk or not, regardless of whether it is unmanaged or managed.

Reply:
turns out unmanaged (unorchestrated) replicas may have nulls as some tests demonstrated. we could set a default, but `null` seems a tad more accurate to me, since it's not known / necessary to know whether the replica uses disk or not.

Reply:
Ah, gotcha. I generally dislike when test features leak into public interfaces, but in this case allowing `NULL` here is strictly more general, and it'll be easier to make this not-nullable in the future than to have it not-nullable now and try to make it nullable in the future.